main/tests/cypress/cypress/integration/project/grid/column/edit-cells/common-transforms/to-text.spec.js

```diff
@@ -0,0 +1,25 @@
+describe(__filename, function () {
+  it('Ensure number is converted to text', function () {
+    cy.loadAndVisitProject('food.mini');
+
+    cy.columnActionClick('NDB_No', [
```

Comment:
Can we use `cy.castColumnTo` here as defined in https://github.com/OpenRefine/OpenRefine/blob/b5192327495aaee18c64a8a0eb8f70aae5adadb7/main/tests/cypress/cypress/support/commands.js#L101 ? Else the tests looks good to me!

Reply:
I have removed cy.columnActionClick() and used cy.castColumnTo() to cast the column to number before testing to-text using cy.columnActionClick().